keylime/revocation_notifier.py

```diff
@@ -159,6 +159,8 @@ def worker_webhook(tosend: Dict[str, Any], url: str) -> None:
 
             time.sleep(next_retry)
 
+        session.close()
```

Comment:
I think that a bit more correct fix is to use the context manager: https://requests.readthedocs.io/en/latest/user/advanced/#session-objects

Reply:
Updated.